cypress/e2e/userInformation.cy.js

```diff
@@ -0,0 +1,127 @@
+describe('Landing page after logging in spec and clicking user information button', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'http://localhost:3001/api/v1/sessions', {
+      statusCode: 200,
+      body: 
+        {token: "The token",
+        user: {
+          data: {
+            id: 2,
+            type: "user",
+            attributes: {
+              name: "Billy",
+              email: "billy@email.com",
+              companies: []
+            }
+          }
+        }
+      },
+    }).as('postUserInfo');
+
+    cy.visit('http://localhost:3000/')
+    cy.get('#email').type('dollyP@email.com');
+    cy.get('#password').type('Jolene123');
+    cy.get('.login-btn').click();
+    cy.wait('@postUserInfo');
+    cy.get('[data-testid="update-user"]').click();
+  });
+
+  it('Should open the user info page;', () => {
+    cy.url().should('include', '/userInformation');
+
+    cy.get('[data-testid="name-input"]').should('have.value', 'Billy');
+    cy.get('[data-testid="email-input"]').should('have.value', 'billy@email.com');
+    cy.get('[data-testid="password-input"]').should('contain', '');
+    cy.get('[data-testid="password-confirmation-input"]').should('contain', '');
+  });
+
+  it('Should allow the user to fill the form and submit it', () => {
```

Comment:
can you be explicit about which form this is?

`Should allow the user to fill the "user info" form and submit it`

Reply:
Adjusted the test comment to match suggestion